locales/en/messages.json

```diff
@@ -1464,10 +1464,10 @@
     "configurationGyroCalOnFirstArm": {
         "message": "Calibrate Gyro on first arm"
     },
-    "configurationDigitalIdlePercent": {
+    "configurationmotorIdle": {
         "message": "Motor Idle (%)"
     },
-    "configurationDigitalIdlePercentHelp": {
+    "configurationmotorIdleHelp": {
         "message": "The Motor Idle value sets the idle speed of the motors when throttle is at minimum position.<br/><br/><strong>Dynamic Idle disabled</strong><br/><br/>The lowest throttle value sent to any motor, while armed, as a percentage of full throttle. Increase it to improve motor startup reliability, to avoid desyncs, and to improve PID responsiveness at low throttle.<br/><br/>Too low: the motors may not start up reliably, or desync at the end of a flip or roll or on hard throttle chops.<br/><br/>Too high: the craft may feel 'floaty'.<br/><br/><strong>Dynamic idle enabled</strong><br/><br/>The maximum throttle allowed, after arming, before takeoff. If RPM is less than dyn_idle_min_rpm, or zero, this throttle value will be sent to the motors. When the motors start spinning, throttle is adjusted to maintain the set RPM, but cannot exceed this value until the quad takes off.<br/><br/>Too low: the motors may not start up reliably.<br/><br/>Too high: the craft may shake on the ground before takeoff."
     },
```

Comment:
maybe 
`The Motor Idle value sets the idle speed of the motors when throttle is 'low' (i.e, when the throttle setpoint is below min_check).` 
`If Dynamic Idle is not enabled, the value sent to the motors will be 'min_command' plus the Motor Idle percentage, for both PWM and DShot ESCs.`
Too low: the motors may not start up reliably, or desync at the end of a flip or roll or on hard throttle chops.
Too high: the craft may feel 'floaty'.  < no changes for these two lines>

`If Dynamic Idle is enabled, and before takeoff, the motor drive value cannot be greater than the Motor Idle percentage above 'min_command'. If the motors do not start up reliably, increase the Motor Idle value to send a higher initial startup value. Once the motors start spinning, Dynamic idle will attempt to attain the set minimum RPM. Note that, before takeoff, the amount of drive is lower at idle than in-flight, due to the Motor Idle limit, and the RPM at idle may be below the configured dyn_idle_min_rpm value. So long as the motors start up reliably, this is OK. To check that your configured dyn_idle_min_rpm value will be achieved in flight, raise throttle above the 'air mode_start_throttle' value, which overcomes the Motor Idle limit.  If the Motor Idle is set too high, and Dynamic Idle is enabled, the craft may shake on the ground before takeoff.`

Sorry for so many words !

Reply:
Is this the longest tooltip :stuck_out_tongue: 

![image](https://github.com/user-attachments/assets/d884699c-a4bc-47eb-a486-d8af05505e9b)

Reply:
OK let's try something shorter :-)
```
Sets the idle speed of the motors while armed and throttle is 'low' (below `min_check`).
**Dynamic Idle not enabled:**
Each motor gets `min_command` plus Motor Idle percent.
Idle too low: motors may not start reliably, spin up slowly, or desync in flips or rolls.
Idle too high: the craft may feel 'floaty'.
Note: analog ESC's must be calibrated so the motors start just above `min_command`.
**Dynamic Idle enabled**
On arming, the 'normal' idle value is sent to each motor, until they spin up.
Once spinning, the motor signal is adjusted to achieve the target RPM.
Before takeoff, the motor signal is limited to the Motor Idle percentage, and the set RPM may not be achieved.  This is OK.  When throttle is raised above the `airmode_motor_start_throttle` percentage, the limit is much higher, and the set RPM should be achieved at idle.
Idle too low: motors may not start reliably
Idle too high: shaking before takeoff (only if dynamic idle is also high)
Note: Dynamic Idle requires DShot and DShot Telemetry.

Reply:
We need the tooltips to be wider :-)
Either that, or the tooltip is responsive to the state of Dynamic idle, and only shows the text for 'not enabled' when it is 'not enabled', and vice versa.
This is perhaps the best solution.